Box.V2/Models/BoxItem.cs

```diff
@@ -73,6 +74,12 @@ public class BoxItem : BoxEntity
         [JsonProperty(PropertyName = FieldModifiedAt)]
         public DateTime? ModifiedAt { get; private set; }
 
+        /// <summary>
+        /// Trashed at timestamp for item
```

Comment:
Match the API docs verbiage...
```suggestion
        /// The time at which this item was put in the trash.
```